tests/RobotFramework/tests/plugin_apt/filter_packages_list_output.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:software    theme:plugins
+
+
+*** Test Cases ***
+Apply name filter
+    ${packages_list}=    Execute Command
+    ...    sudo /etc/tedge/sm-plugins/apt list --name tedge
+    ...    exp_exit_code=0
+    Should Match Regexp    ${packages_list}    ^tedge\\s+${VERSION}
+
+Apply maintainer filter
+    ${packages_list}=    Execute Command
+    ...    sudo /etc/tedge/sm-plugins/apt list --maintainer thin-edge.*
+    ...    exp_exit_code=0
+    Should Match Regexp    ${packages_list}    c8y.*${VERSION}(.|\\n)*tedge\\s+${VERSION}
+
+Apply both filters
+    ${packages_list}=    Execute Command
+    ...    sudo /etc/tedge/sm-plugins/apt list --name sudo --maintainer thin-edge.*
```

Comment:
Why the name is `sudo`? Using it as an example package name which doesn't exist?

Reply:
Actually, quite the opposite. Filters work here like union so I wanted to show that when we use it with the maintainer filter set to thin-edge, it will print not only all the tedge packages but also sudo package. And I think we can safely assume that the sudo package will be available (In fact, it is even used when executing the test command).